### DISCUSSION TITLE
Povratna informacija na prijavu projekta

### DISCUSSION THREAD

**Discussion by boristomas** [foivz/pi22-aslavic-lstrunjak-hcolakovi-azivko#1] [Announcements]
Poštovani kolege,

oznaka vašeg tima je PI2229
Prijava je u redu, ali:
F02 nije funkcionalnost.
F06 i F05 su jedna funkcionalnosti.
F07 nema smisla. - trebalo bi to pridružiti nekoj funkcionalnosti gdje se to već koristi.

Lijep pozdrav,
Boris

**Comment by HasanColakovic99** [comment]
Lijep pozdrav profesore,

evo mi smo prepravili greške te dodali neke nove funkcionalnosti za našu aplikaciju, te bismo voljeli da dobijemo povratnu informaciju kako bismo mogli krenuti sa realizacijom prve faze.

Unaprijed se zahvaljujemo,
Tim PI2229.

**Reply by boristomas** [reply]
a ono... F10 F07 F06 F05 su jedna funkcionalnost, to je jako teško razdvojiti.

**Comment by HasanColakovic99** [comment]
Lijep pozdrav profesore,

evo ponovo smo promijenili/dodali funkcionalnosti pa ako nam možete dati povratnu informaciju kako bismo mogli nastaviti sa izradom.

Unaprijed se zahvaljujemo,
Tim PI2229.

**Reply by boristomas** [reply]
u redu je

**Comment by Z1V3** [comment]
Lijep pozdrav profesore,

evo samo da javimo da smo ažurirali funkcionalnosti kako smo dogovorili danas na konzultacijama. Rekli ste da maknemo funkcionalnosti broj 6,7,11 i 12, te da ažuriramo i ko će koju funkcionalnost obraditi.

Unaprijed se zahvaljujemo,
Tim PI2229.
